index.js

```diff
@@ -152,38 +152,43 @@ module.exports = {
             if (err) return cb(err)
             if (isFollowing) { return cb(new Error('already following')) }
 
-            // although we already know the current feed
-            // it's included so that request cannot be replayed.
-            if (!req.feed) { return cb(new Error('feed to follow is missing')) }
+            server.friends.isBlocking({ source: server.id, dest: req.feed }, (err, isBlocking) => {
+               if (err) return cb(err)
+               if (isBlocking) { return cb(new Error('user is blocked')) }
 
-            if (invite.used >= invite.total) { return cb(new Error('invite has expired')) }
+              // although we already know the current feed
+              // it's included so that request cannot be replayed.
+              if (!req.feed) { return cb(new Error('feed to follow is missing')) }
```

Comment:
Could we move this up before the isBlocking call? Doesn't seem to be valid to call isBlocking when the dest is undefined.